my-app/src/app/modules/courses-page/courses-page.component.ts

```diff
@@ -27,14 +29,16 @@ export class CoursesPageComponent implements OnInit {
   }
 
   public onDeleteCourse(courseId: number): void {
-    const courseTitle: string | undefined = this.coursesHandlerService.getItemById(courseId)?.title;
-
-    if (confirm(
-      `Are you sure to delete '${courseTitle}' course`) && this.visibleCourses
-    ) {
-      this.visibleCourses = this.coursesHandlerService.removeItem(courseId);
-      console.log('Course has been deleted successfully')
-    }
+    const foundCourse: ICourse | undefined = this.coursesHandlerService.getItemById(courseId);
+
+    if (foundCourse) {
+      if (confirm(
```

Comment:
Instead of
```
if(a) {
  if(b) {...}
} else {}
```
try 
```
if (!a) {...}
if (b) {...}
```
It makes the structure more flat and easy to understand.

Reply:
Indeed, fixed, thank you